<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/ICON.ico</normaloff>:/images/ICON.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="openglwindow" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>961</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1092</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>模型载入</string>
    </property>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>模型选择</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>纹理</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>光照</string>
    </property>
    <addaction name="action_12"/>
    <addaction name="action_14"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>帮助(?)</string>
    </property>
    <addaction name="action_15"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_2">
   <property name="text">
    <string>魔方</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>大理石</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>陶瓷</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>点模型</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>线框模型</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>面模型</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>旋转
</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>缩放</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>打开...
</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>保存为...</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>点光源</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>平行光源</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>使用手册</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>openglwindow</class>
   <extends>QWidget</extends>
   <header>openglwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
